x-pack/agent/docs/agent_configuration_example.yml

```diff
@@ -0,0 +1,209 @@
+outputs:
+  default:
+    api_key: VuaCfGcBCdbkQm-e5aOx:ui2lp2axTNmsyakw9tvNnw
+    hosts: ["localhost:9200"]
+    ca_sha256: "7lHLiyp4J8m9kw38SJ7SURJP4bXRZv/BNxyyXkCcE/M="
+    # Not supported at first
+    queue:
+      type: disk
+
+  long_term_storage:
+    api_key: VuaCfGcBCdbkQm-e5aOx:ui2lp2axTNmsyakw9tvNnw
+    hosts: ["localhost:9200"]
+    ca_sha256: "7lHLiyp4J8m9kw38SJ7SURJP4bXRZv/BNxyyXkCcE/M="
+    queue:
+      type: disk
+
+  monitoring:
+    api_key: VuaCfGcBCdbkQm-e5aOx:ui2lp2axTNmsyakw9tvNnw
+    hosts: ["localhost:9200"]
+    ca_sha256: "7lHLiyp4J8m9kw38SJ7SURJP4bXRZv/BNxyyXkCcE/M="
+
+settings.monitoring:
+  use_output: monitoring
+
+datasources:
+  # use the nginx package
+  - id: nginx-x1
+    title: "This is a nice title for human"
+    # Package this config group is coming from. On importing, we know where it belongs
+    # The package tells the UI which application to link to
+    package:
+      name: epm/nginx
+      version: 1.7.0
+    namespace?: prod
+    constraints:
+      # Contraints look are not final
+      - os.platform: { in: "windows" }
```

Comment:
this should be ecs compliant?

Reply:
YES!

Reply:
Yes it should, I should that that the constraints look is not final. but lest make sure the definition of them both work in the Agent and in the UI. It would be good to leverage that information to display more useful data to the user when they visualize and agent or a configuration.